### DISCUSSION TITLE
Is there any learning resources about the internal architecture/design about the reactive mechanism?

### DISCUSSION THREAD

**Discussion by zhaoyao91** [solidjs/solid#378] [Q&A] [answered]
Solid's reactive things are great and magic. I tried to read about the source code, felt hard to grab the high level design and purpose of what the code is.

The resources I can found in the readme are most about the usage of solid. Is there any article/blog about the internal things?

Thanks 福

**Comment by ryansolid** [comment, marked as answer]
I've written dozen's of articles on Solid you can find here: https://github.com/ryansolid/solid/blob/master/documentation/resources/articles.md#latest-articles

The Designing Solid series talks about design decisions. If you want a walkthrough of the code I fear I don't have that. Just articles talking about choices.
